src/client/kube_store/index.ts

```diff
@@ -233,6 +48,47 @@ export default class KubeStore {
     return this.instance || (this.instance = new this(''));
   }
 
+  public patchResource(resource: KubeResource, name: string, patch: object): object {
+    let source: object = {};
+    let key: string = '';
+    const myPatch: object = {'foo':'bar'};
```

Comment:
Don't think we want dummy data here.

Reply:
+1 remove this and just use 'patch' object that is passed in